app/src/ui/changes/files-changed-badge.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react'
+
+interface IFilesChangedBadgeProps {
+  readonly numFilesChanged: number
+}
+
+interface IFilesChangedBadgeState {
```

Comment:
We can get away without relying on state. Using props should be sufficient for this component.